css/offline.less

```diff
@@ -26,6 +26,14 @@
             }
         }
 
+        & > #offline-banner {
```

Comment:
Would a CSS class do here, and without the '>'?
Keeping selectors to the least specific possible means things are easier to override, and keeps CSS more maintainable.

Reply:
Yep, change made